.github/workflows/check_db_migration_confict.yml

```diff
@@ -0,0 +1,59 @@
+name: Check DB migration conflict
+on:
+  push:
+    paths:
+      - "superset/migrations/**"
+
+jobs:
+  check_db_migration_conflict:
+    name: Check DB migration conflict
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v2
+      - name: Check and notify
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm not sure secrets will be available on `push` events from forked repositories.

Reply:
I think `push` events only triggers for the base repo. Forked repos will run the `push` event with their own secret, which does not have access to the base repo---in this case, the forked repo is considered its own base repo.

Either way, it shouldn't matter because we only want to run the workflow in the base branch anyway.

Reply:
The `GITHUB_TOKEN` provided to forks is read-only. I believe a write token is required to publish a comment on a PR.

Reply:
Yep, the confusion was related to `push` events running on PRs from forked repos, but looking at recent action runs, this doesn't appear to be the case. The GitHub Actions docs leave something to be desired.